kano_settings/config_file.py

```diff
@@ -41,14 +41,14 @@ def merge_dicts(base, override):
     'Keyboard-continent-index': 1,
     'Keyboard-country-index': 21,
     'Keyboard-variant-index': 0,
-    'Keyboard-continent-human': 'america',
-    'Keyboard-country-human': 'United States',
-    'Keyboard-variant-human': 'Generic',
-    'Audio': 'Analogue',
+    'Keyboard-continent-human': _("america"),
```

Comment:
It does indeed cause a problem, interestingly when the package is being installed:
```
The following extra packages will be installed:
  kano-connect
The following packages will be upgraded:
  kano-connect kano-settings
2 upgraded, 0 newly installed, 0 to remove and 12 not upgraded.
Need to get 524 kB of archives.
After this operation, 62.5 kB of additional disk space will be used.
Get:1 http://dev.kano.me/archive-jessie/ scratch/main kano-settings armhf 3.2.0-1.20160922 [274 kB]
Get:2 http://dev.kano.me/archive-jessie/ scratch/main kano-connect armhf 3.2.0-1.20160922 [250 kB]
Fetched 524 kB in 0s (2,577 kB/s) 
(Reading database ... 91400 files and directories currently installed.)
Preparing to unpack .../kano-settings_3.2.0-1.20160922_armhf.deb ...
Unpacking kano-settings (3.2.0-1.20160922) over (3.2.0-1.20160914) ...
insserv: Service kano-settings has to be enabled to start service kano-reboot-clear
insserv: exiting now!
update-rc.d: error: insserv rejected the script header
Preparing to unpack .../kano-connect_3.2.0-1.20160922_armhf.deb ...
Unpacking kano-connect (3.2.0-1.20160922) over (3.2.0-1.20160914) ...
Processing triggers for systemd (215-17+deb8u5) ...
Processing triggers for kano-apps (2.2.0-1.20160818) ...
Processing triggers for desktop-file-utils (0.22-1) ...
Processing triggers for mime-support (3.58) ...
Setting up kano-connect (3.2.0-1.20160922) ...
Setting up kano-settings (3.2.0-1.20160922) ...
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/lib/python2.7/dist-packages/kano_settings/system/advanced.py", line 30, in <module>
    from kano_settings.config_file import get_setting, set_setting
  File "/usr/lib/python2.7/dist-packages/kano_settings/config_file.py", line 44, in <module>
    'Keyboard-continent-human': _("america"),
NameError: name '_' is not defined
```